region/maridia/inner-pink/East Cactus Alley Room.json

```diff
@@ -871,6 +871,159 @@
+              "maxHeight": 2,
+              "minTiles": 45,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canInsaneJump",
+                "canMomentumConservingMorph"
+              ],
+              "note": ["This applies to Statues Hallway and Baby Kraid Room."],
+              "devNote": [
+                "This almost works from Metroid Room 1 (with $4.1 run speed) but comes up a few pixels short."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 21.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "environment": "water",
+              "requires": [
+                "canTrickyDashJump",
+                "canInsaneJump",
+                "canGravityJump",
+                "canMomentumConservingMorph"
+              ],
+              "note": [
+                "This applies to Draygon's Room.",
+                "Start a little less than a tile away from the wall, gain run speed, and time a pause to unequip Gravity after Samus jumps and aims down;",
+                "hold down through the unpause to buffer a morph."
+              ],
+              "detailNote": [
+                "Gain extra run speed of $4.0 or $4.1."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 41.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 0]],
+              "requires": [
+                "canInsaneJump",
+                "canMomentumConservingMorph"
+              ],
+              "note": [
+                "This applies to Blue Brinstar Energy Tank Room (Power Bomb blocks broken), Bowling Alley (middle), and Basement."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 45,
+              "speedBooster": true,
+              "obstructions": [[5, 0]],
+              "requires": [
+                "canInsaneJump",
+                "canMomentumConservingMorph",
+                "canInsaneMidAirMorph"
+              ],
+              "note": [
+                "This applies to Basement (power off)."
+              ]
+            },
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 39.4375,
+              "speedBooster": true,
+              "obstructions": [[3, 2]],
+              "requires": [],
+              "note": ["This applies to Metal Pirates Room."]
+            },            
+            {
+              "minHeight": 3,
+              "maxHeight": 3,
+              "minTiles": 39.4375,
+              "speedBooster": true,
+              "obstructions": [[5, 2]],
+              "requires": [
+                {"or": [
+                  "canMomentumConservingTurnaround",
```

Comment:
I can't get the turnaround one to work.

Reply:
I never got a turnaround to work either. Must have been a copy-paste error. Fixed now.